src/browsers/schema/content/homepage.md

```diff
@@ -0,0 +1,9 @@
+Examples - Gene: [SETD1A](link), Variant: [16-30992057-CAG-C](link)
+
+The Schizophrenia Exome Sequencing Meta-analysis (SCHEMA) consortium is a large multi-site collaboration dedicated to aggregating, generating, and analyzing high-throughput sequencing data of schizophrenia patients to improve our understanding of disease architecture and advance gene discovery.  
+
+The Phase I data set (GRCh37/hg19) contains the analysis of exomes from 24,248 cases and 97,322 controls, and _de novo_ mutations from 3,402 parent-proband trios. These results have pinpointed genome-wide significant results associating ultra-rare protein-coding variants in individual genes to risk of schizophrenia.  
+
+We would also like to thank the many tens of thousands of patients and families who generously contributed to our effort. This project is made possible by the generosity of many funders, including the [Stanley Foundation](https://www.broadinstitute.org/stanley), and [National Institutes of Mental Health (NIMH)](https://www.nimh.nih.gov/index.shtml), and the leadership of its members. The contributing groups to this effort are listed [here](about.md), and the terms of use are seen [here](terms.md).
```

Comment:
Can we reword the last sentence so that the link text contains some information about what the link is referring to? Having "here", etc as the link text is inaccessible to people using screen readers.

Reply:
Rephrased sentence.